os/include/tinyara/pm/pm.h

```diff
@@ -313,6 +307,12 @@ enum pm_timer_type_e {
 	/* Scope for future expansion, up to 8 timer types can be supported */
 };
 
+/* This structure is used to send data to pm driver from app side */
+struct pm_arg_s {
+	enum pm_domain_e pm_domain;
+	enum pm_state_e pm_state;
+};
+
```

Comment:
It will be used a lot from apps

how about `typedef pm_arg_t struct pm_arg_s` ?

Reply:
done